5 individual article

Jord000 opened the pull request:
component build to show each individual article from a news feed style thread

Keb84 commented on __tests__/app.test.js:
```diff
@@ -54,10 +54,38 @@ describe('GET:200 /api', () => {
+        articles.forEach((article) => {
+          expect(Object.keys(article)).toEqual(
+            expect.arrayContaining(correctProperties)
+          )
```
You should also be testing the values of the keys. You could use a jest match like toMatchObject and expect.any(string) for example to make it more dynamic.

Jord000 replied:
added thanks!

Keb84 reviewed (approved):
Nice work. Please see my comment to add to your test, make the changes and merge when ready 🙂

Jord000 committed:
fixed error in console on main feed removed console logs

Jord000 committed:
truncate text and grids improved on main feed button added for more detail

Jord000 committed:
error to login
styling for all pages so far to help navigate
page for individual article but no return button
dynamic endpoints created

Jord000 committed:
nav bar to get back to feed added

Jord000 committed:
set so without login myfeed defaults to login
nav bar works better now and is setup for further detail later

nc-nick-jones reviewed (changes requested):
Nice work jordan..  I think that maybe your navigation to the single article page could be simpler.  Please can we have a single route /articles/:article_id and a URL that looks this way for all single articles?  Instead of useNavigate, and useSearchParams, this can be achieved more cleanly with Link and useParams().  Currently your id is looking like a query, which I don't think we want.   Nice work so far 👍

Jord000 committed:
improved routing method using link to get parametric endpoint

nc-ali-abdallah reviewed (approved):
Nice looking good, I'd rethink having a button a user has to press to first enter the site it's not very good UX.

Great job implementing feedback on the routing.

Very nice responsive design looks good on all screen sizes.
Merge when ready.
